### PR TITLE
Update container-runtime.md - Troubleshoot docs

### PR DESCRIPTION
**Pull request by edgarlanting**
Add more clarification to how this analyzer functions. This is also a documentation improvement request from this [issue](https://github.com/replicated-collab/settlemint-replicated/issues/44)

### CODE REVIEW

File: docs/source/analyze/container-runtime.md [hunk 1]
```diff
@@ -9,6 +9,26 @@ The `when` attribute supports standard comparators to compare to the detected ru
...
+"nodeInfo": {
+  "containerRuntimeVersion": "docker://20.10.5",
+```
+
+The value for `containerRuntimeVersion` can also be retrieved by manually running the following command:
+`kubectl get node [nodename] --no-headers -o=jsonpath='{.status..nodeInfo.containerRuntimeVersion}'`
+
+As an example, the output will look like the following: `containerd://1.6.8`
+
+Troubleshoot uses a url parser to retrieve the scheme from the runtime. This is referenced in the Troubleshoot code at: https://github.com/replicatedhq/troubleshoot/blob/d5c7522be50d7e310d05e7a9679183f4056a89f5/pkg/analyze/container_runtime.go#L116)
```
Comment (banjoh) [inline docs/source/analyze/container-runtime.md:25]: Referencing code in documentation can be burdensome at times. We need to ensure the documentation keeps up with the code whenever changes are made. Do we really need to point where the logic resides in the docs? Perhaps we could document `compareRuntimeConditionalToActual` function instead to help any interested people find the implementation? Adding some keywords in the [doc comment](https://tip.golang.org/doc/comment) might go a long way when someone searches/greps the repo.

File: docs/source/analyze/container-runtime.md [hunk 2]
```diff
@@ -9,6 +9,26 @@ The `when` attribute supports standard comparators to compare to the detected ru
 The `containerRuntime` analyzer uses data from the [clusterResources collector](https://troubleshoot.sh/collect/cluster-resources).
 The `clusterResources` collector is automatically added and will always be present.
 
+The `containerRuntime` analyzer is based on the `containerRuntimeVersion` field that is available on each Kubernetes node. 
+This is reflected in an (**example**) support bundle under `cluster-resources/nodes.json` as:
```
Comment (banjoh) [inline docs/source/analyze/container-runtime.md:13]: MINOR: I'd leave out the text highlight from the word `example`

File: docs/source/analyze/container-runtime.md (hunk 1 above)
Comment (edgarlanting) [reply docs/source/analyze/container-runtime.md:25]: That is actually a very good point to mention @banjoh Let me review my PR to do some rewriting where needed.

**Review by banjoh — DISMISSED** [review]

**Comment by edgarlanting** [comment]
Thanks @jonquil2002

**Comment by edgarlanting** [comment]
@jonquil2002 its all good I'm done with this PR

**Pull request merged**